secretcache/cacheVersion.go

```diff
@@ -75,7 +73,7 @@ func (cv *cacheVersion) executeRefresh(ctx context.Context) (*secretsmanager.Get
 		SecretId:  &cv.secretId,
 		VersionId: &cv.versionId,
 	}
-	return cv.client.GetSecretValueWithContext(ctx, input, request.WithAppendUserAgent(userAgent()))
```

Comment:
How does the User Agent get appended now?

Reply:
Added the user agent into the middleware when the client gets created